server/conf.js

```diff
@@ -0,0 +1,8 @@
+const  mysql = require('mysql');
+const  connection = mysql.createConnection({
+host :  'localhost', // adresse du serveur
+user :  'root', // le nom d'utilisateur
+password :  'FEVZNfevzn1.', // le mot de passe
+database :  'tonton_sommelier', // le nom de la base de données
+});
+module.exports = connection;
```

Comment:
don't push this... and change your password ;)

Reply:
we have had this file to gitignore, why it was send ?